player/multiview/js/script.js

```diff
@@ -0,0 +1,271 @@
+}
+
+function getPlayerInstance(playerConfig, source) {
+  let player = findPlayerForSource(source);
+  if (player) {
+    // An active player instance already exists for this source
+    return player;
+  }
+
+  player = unusedPlayers.shift();
+  if (player) {
+    // Re-use one of the unused player instances
+    player.load(source);
+    return player;
+  }
+
+  // Create a new player instance
+  const container = createPlayerTile();
+  const newPlayer = new bitmovin.player.Player(container, playerConfig);
+  container.addEventListener('click', event => onTileClicked(container, newPlayer, event), true);
+
+  newPlayer.load(source).then(() => {
+    // Move the control bar to the controlbars-container to be full-width
+    // This lets us simulate a common control bar for all players
+
+    const controlBar = container.getElementsByClassName('bmpui-ui-controlbar')[0];
+    controlBar.setAttribute('data-player-id', getPlayerId(newPlayer).toString());
+    if (source === primarySource) {
+      controlBar.classList.add('active');
+    }
+
+    controlBarsContainer.appendChild(controlBar);
+  });
+  allPlayers.push(newPlayer);
+
+  return newPlayer;
+}
+
+function findPlayerForSource(source) {
+  return allPlayers.find(player => player.getSource() === source);
+}
+
+function setPrimaryPlayer(newPrimaryPlayer, newPrimarySource) {
+  if (newPrimarySource === primarySource || newPrimaryPlayer === null) {
+    return;
+  }
+
+  const container = document.getElementById('player-container');
+
+  // Remove active classes from the previous primary player
+  container.querySelector('.bmpui-ui-controlbar.active')?.classList.remove('active');
+  container.querySelector('.tile.primary')?.classList.remove('primary');
+
+  if (newPrimaryPlayer == null) {
+    primarySource = null;
+    return;
+  }
+
+  // Add active classes to the new primary player
+  newPrimaryPlayer.getContainer().classList.add('primary');
+  getControlBar(newPrimaryPlayer)?.classList.add('active');
+
+  const previousPrimaryPlayer = findPlayerForSource(primarySource);
+  if (previousPrimaryPlayer) {
+    // Keep the volume state consistent across players
+    if (previousPrimaryPlayer.isMuted()) {
+      newPrimaryPlayer.mute();
+    } else {
+      newPrimaryPlayer.unmute();
+    }
+    const volume = previousPrimaryPlayer.getVolume();
+    newPrimaryPlayer.setVolume(volume);
+    previousPrimaryPlayer.mute();
+  }
+
+  primarySource = newPrimarySource;
+}
+
+function createPlayerTile() {
+  const tile = document.createElement('div');
+
+  tile.classList.add('tile');
+  tile.id = 'player';
+  tile.draggable = true;
+
+  tile.addEventListener("dragstart", event => {
+    if (event.target.id === 'player') {
+      draggedElement = event.target;
+    }
+  });
+  tile.addEventListener("dragover", (event) => {
+    event.preventDefault();
+  });
+  tile.addEventListener("drop", (event) => {
```

Comment:
Please change the `"` to `'` since these are the only places the double quotes are used... No idea why this annoys me that much :D

Reply:
lol, done :D